commands/user.bee.inc

```diff
@@ -98,14 +196,213 @@ function users_bee_callback($arguments, $options) {
+    $account->save();
+    bee_message(bt("User '!user' has been created, with password !pass", array(
+      '!user' => $arguments['username'], '!pass' => $password,
+    )), 'success');
+  }
+  else {
+    bee_message(bt("Could not create a new user account with the name, !user", array(
+      '!user' => $name,
+    )), 'error');
+  }
+}
+
+/**
+ * Command callback: Delete/cancel a given user.
+ */
+function user_cancel_bee_callback($arguments, $options) {
+  // Load the user account.
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' could not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+  $answer = bee_confirm(bt("Are you sure to delete the user !user?", array(
+    '!user' => $arguments['username'],
+  )), FALSE);
+  if (!$answer) {
+    return;
+  }
+
+  user_cancel(array(
+    'user_cancel_notify' => TRUE,
+    'user_cancel_method' => 'user_cancel_delete',
+  ), $user->uid, 'user_cancel_delete');
+
+  // Invoke the Batch API, disable backdrop_goto(), and execute.
+  $batch = &batch_get();
+  $batch['progressive'] = FALSE;
+  batch_process();
+
+  bee_message(bt("User account '!user' has been removed.", array(
+    '!user' => $user->name,
+  )), 'success');
+}
+
+/**
+ * Command callback: Block a given user.
+ */
+function user_block_bee_callback($arguments, $options) {
+  // Load the user account.
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' could not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+
+  $user->status = 0;
+  user_save($user);
+
+  bee_message(bt("User '!user' has been blocked.", array(
+    '!user' => $user->name,
+  )), 'success');
+}
+
+/**
+ * Command callback: Unblock a given user.
+ */
+function user_unblock_bee_callback($arguments, $options) {
+  // Load the user account.
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' could not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+
+  $user->status = 1;
+  user_save($user);
+
+  bee_message(bt("User '!user' has been unblocked.", array(
+    '!user' => $user->name,
+  )), 'success');
+}
+
+/**
+ * Command callback: Add a role to a given user.
+ */
+function user_add_role_bee_callback($arguments, $options) {
+  $user = user_load_by_name($arguments['username']);
+  if (!$user) {
+    bee_message(bt("User account '!user' could not be found.", array(
+      '!user' => $arguments['username'],
+    )), 'error');
+    return;
+  }
+  $roles = user_roles(TRUE);
+  if (empty($roles[$arguments['role']])) {
+    if ($roles['authenticated']) {
+      unset($roles['authenticated']);
+    }
+    $rolestxt = implode(",", $roles);
+      bee_message(bt("The role '!role' could not be found. Valid roles available: !roles", array(
```

Comment:
We had two different ways of phrasing this between the add role function and remove role function. This is clearer.

```suggestion
      bee_message(bt("The '!role' role could not be found. Valid roles available: !roles", array(
```